ldes-server-infra-mongo/mongo-retention-repository/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/retention/MemberPropertiesRepositoryImpl.java

```diff
@@ -51,7 +51,9 @@ public MemberPropertiesRepositoryImpl(MemberPropertiesEntityRepository memberPro
 
 	@Override
 	public void insertAll(List<MemberProperties> memberProperties) {
-		final List<MemberPropertiesEntity> entities = memberProperties.stream()
+		final List<MemberPropertiesEntity> entities = memberProperties
+				.stream()
+				.parallel()
```

Comment:
if you're just a parallelising just to merge it back into a list, you're missing perfo gain.. would remove the parallel here